edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -48,16 +48,19 @@
 
   {% if is_course_staff %}
   <script type="text/template" id="sga-grading-tmpl">
-    <table class="gridtable">
+    <table class="gridtable tablesorter" id="submissions">
+      <thead>
       <tr>
-        <th>Username</th>
-        <th>Name</th>
-        <th>Filename</th>
-        <th>Uploaded</th>
-        <th>Grade</th>
-        <th>Instructor's comments</th>
-        <th>Annotated</th>
+        <th class="header">Username <i class="icon fa fa-sort"/></th>
+        <th class="header">Name <i class="icon fa fa-sort"/></th>
+        <th class="header">Filename <i class="icon fa fa-sort"/></th>
+        <th class="header">Uploaded <i class="icon fa fa-sort"/></th>
+        <th class="header">Grade <i class="icon fa fa-sort"/></th>
+        <th class="header">Instructor's comments <i class="icon fa fa-sort"/></th>
```

Comment:
Just an observation: At first I wondered why anyone would sort on the comment field, but then I considered that an instructor could preface comments with a tag or their initials, making a sorted comment field useful.